repo/packages/C/chronos/7/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "2.5.1",
   "scm": "https://github.com/mesos/chronos.git",
   "maintainer": "support@mesosphere.io",
-  "description": "A fault tolerant job scheduler for Mesos which handles dependencies and ISO8601 based schedules.",
+  "description": "A fault tolerant job scheduler for Mesos which handles dependencies and ISO8601 based schedules. [DEPRECATED]",
```

Comment:
not sure if that's a pattern we already have in use. if so, that's fine with me. if not, i'd vote to make that more prominent. 

how about: 
```
  "description": "[DEPRECATED] A fault tolerant job scheduler for Mesos which handles dependencies and ISO8601 based schedules.",
  ...

  "preInstallNotes": "The use of chronos has been DEPRECATED! If you still choose to use it, we recommend a minimum of one node with at least 1 CPU and 2GB of RAM available.",

```

Reply:
I had seen with other packages that some did it with the pattern I used, but I like your suggestions.